hw/ip/otbn/util/rig/README.md

```diff
@@ -0,0 +1,4 @@
+# OTBN Random Instruction Generator
```

Comment:
I think there's probably mileage in expanding this a bit to explain the different parts and how they interact. There are plenty of comments in the code but some kind of roadmap would be helpful.

Reply:
Yeah, good point. Are you happy for this to be in a follow-up patch (rather than make this one even bigger!)? If so, I can try to push that up this afternoon.

Reply:
Actually, I just re-read that file and... yes. I think I intended to fill it out a bit more! So I've just done so. I'm definitely imagining that the documentation will expand over time, but I'd rather do that after a bit more functionality is in place, rather than have to rewrite it as we change our minds about how things should work.